javascript/test/ParserTest.ts

```diff
@@ -308,5 +309,128 @@ description
 
       assert.strictEqual(pickle.steps[0].argument.docString.content, '```what')
     })
+
+    it("parses Markdown data tables with headers", () => {
+      const markdown = `## Feature: DataTables
+
+### Scenario: minimalistic
+
+* Given a simple data table 
+  | foo | bar |
+  | --- | --- |
+  | boz | boo |
+`
+      const ast = parser.parse(markdown)
+      const gherkinDocument: messages.GherkinDocument = {
+        "comments": [
+          {
+            "location": {
+              "column": 3,
+              "line": 7
+            },
+            "text": undefined
```

Comment:
This attribute is required, but conflicts with the Python implementation (see below)